py3status/modules/weather_owm.py

```diff
@@ -0,0 +1,708 @@
+# -*- coding: utf-8 -*-
+'''
+Ultimately customizable weather module based on the IP-API Geolocation API
```

Comment:
This can be ignored for this PR imo. If it becomes a problem, we will address it later.